### PR TITLE
[bitnami/thanos] Additional labels for Prometheus ServiceMonitor

### PR DESCRIPTION
**Pull request by thesolution90**
**Description of the change**

Possibility to add own custom labels to ServiceMonitor objects in order to enable service discovery for already deployed Prometheus Operators.

**Benefits**

Service Discovery enabled for custom Prometheus Operator

**Possible drawbacks**

No drawbacks known

**Applicable issues**

  - fixes #

**Additional information**

**Checklist** 
- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [X] Variables are documented in the README.md
- [X] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)
- [X] If the chart contains a `values-production.yaml` apart from `values.yaml`, ensure that you implement the changes in both files

:warning: Keep in mind that if you want to make changes to the kubeapps chart, please implement them in the [kubeapps repository](https://github.com/kubeapps/kubeapps/tree/master/chart/kubeapps). This is only a synchronized mirror.

### CODE REVIEW

File: bitnami/thanos/templates/bucketweb/servicemonitor.yaml [hunk 1]
```diff
@@ -8,6 +8,9 @@ metadata:
   {{- end }}
   labels: {{- include "thanos.labels" . | nindent 4 }}
     app.kubernetes.io/component: bucketweb
+{{- if .Values.metrics.serviceMonitor.labels }}
+{{ toYaml .Values.metrics.serviceMonitor.labels | indent 4}}
```
Comment (javsalgar) [inline bitnami/thanos/templates/bucketweb/servicemonitor.yaml:12]: Could you use `{{-` and `nindent` instead? With that you could indent this block to the same level as the rest of the labels, making the code more readable

File: bitnami/thanos/Chart.yaml [hunk 2]
```diff
@@ -15,6 +15,6 @@ maintainers:
 name: thanos
 sources:
   - https://github.com/bitnami/bitnami-docker-thanos
-version: 1.3.4
+version: 1.3.5
```
Comment (javsalgar) [inline bitnami/thanos/Chart.yaml:18]: As it is adding a feature, I would bump the minor version instead

File: bitnami/thanos/templates/bucketweb/servicemonitor.yaml (hunk 1 above)
Comment (thesolution90) [reply bitnami/thanos/templates/bucketweb/servicemonitor.yaml:12]: Yeah makes sense. Will change it

File: bitnami/thanos/Chart.yaml (hunk 2 above)
Comment (thesolution90) [reply bitnami/thanos/Chart.yaml:18]: Will change it to 1.4.0

**Pull request merged**